templates/authorizing_functionaries.html

```diff
@@ -0,0 +1,82 @@
+{#-#################################################################
+<File Name>
+  authorizing_functionaries.html
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Apr 06, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Shows multi select widgets using select2 JS plugin that can be used
+  to authorize functionaries for steps and set a threshold, i.e. how many
+  functionaries need to provide link metadata for a step.
+
+#################################################################-#}
+
+{% import '_macros.html' as macros %}
+{% extends "base.html" %}
+
+{#- BEGIN: Rendered content -#}
+{% block content %}
+  {{ macros.progress_bar(value=75) }}
+  <hr>
+  <h1>Who is authorized to do what in your Project?</h1>
```

Comment:
What am I supposed to enter in these input fields?  The label says,
"authorize functionaries for this step in select box below."  Maybe include an example.  When I click on the input field I get, "No results found."  Maybe it's because my upload of the public key failed...

Reply:
Will fix with #19 and #26.